model_managers/model_manager.py

```diff
@@ -386,7 +386,7 @@ def delete_relationship(self, rel_list: [[str, str, str]], identifier='label'):
 
         q = f"""
         UNWIND $rels as rel
-        WITH rel[0] as from, rel[1] as to, rel[2] as type
+        WITH rel[0] as from, rel[1] as to, rel[3] as type
```

Comment:
Reordering of the items within the create_rel/delete_rel caused a number of other functions return unexpected outputs. When introducing a change like this , please only add an additional item at the end of the list.

Currently 7 unit tests fail - please update the above and make sure all tests pass.
Thank you

Reply:
My bad. I have made the changes. Thanks!

Reply:
still see a couple of unit tests failing:
FAILED tests/derivation_method/test_derivation_method.py::TestDerivationJson::test_generate_link_actions - KeyError: "Could...
FAILED tests/tests_model_manager/test_mm2.py::test_get_all_ct - AssertionError: assert [{'Codelist C...': None, ...}] == [{...
================================== 2 failed, 183 passed, 384 warnings in 265.68s (0:04:25) ===================================

could you please take a look?
They might accidently pass, but please take care of sorting so that they always pass.
thanks

Reply:
I have fixed test_get_all_ct. There is some issue with compare_method_json() for test_generate_link_actions, seems like it is unable to remove nodes. I have to debug this but don't think so it is because of the changes I made for propagated_from prop. Can we create a bugfix for this in next sprint?